packages/amplify-graphql-transformer-core/src/transformation/transform.ts

```diff
@@ -250,9 +264,113 @@ export class GraphQLTransform {
+        } else if (!_.isEmpty(constructPathObj.nestedStack)) {
+          const fieldType = constructPathObj.nestedStack!.stackType;
+          const fieldName = constructPathObj.nestedStack!.stackName;
+          const resourceName = constructPathObj.resourceName;
+          if (constructPathObj.resourceType.includes('Resolver')) {
+            // console.log("constrcut",constructPathObj);
+            if (amplifyApiObj[fieldType][fieldName]['resolvers']) {
+              amplifyApiObj[fieldType][fieldName]['resolvers'][resourceName] = resource;
+            } else {
+              amplifyApiObj[fieldType][fieldName]['resolvers'] = {};
+              amplifyApiObj[fieldType][fieldName]['resolvers'][resourceName] = resource;
+            }
+          } else if (constructPathObj.resourceType.includes('FunctionConfiguration')) {
+            if (amplifyApiObj[fieldType][fieldName]['appsyncFunctions']) {
+              amplifyApiObj[fieldType][fieldName]['appsyncFunctions'][resourceName] = resource;
+            } else {
+              amplifyApiObj[fieldType][fieldName]['appsyncFunctions'] = {};
+              amplifyApiObj[fieldType][fieldName]['appsyncFunctions'][resourceName] = resource;
+            }
+          } else {
+            if (amplifyApiObj[fieldType]) {
+              if (amplifyApiObj[fieldType][fieldName]) {
+                amplifyApiObj[fieldType][fieldName][resourceName] = resource;
+              } else {
+                amplifyApiObj[fieldType][fieldName] = {};
+                amplifyApiObj[fieldType][fieldName][resourceName] = resource;
+              }
+            } else {
+              amplifyApiObj[fieldType] = {};
+              if (amplifyApiObj[fieldType][fieldName]) {
+                amplifyApiObj[fieldType][fieldName][resourceName] = resource;
+              } else {
+                amplifyApiObj[fieldType][fieldName] = {};
+                amplifyApiObj[fieldType][fieldName][resourceName] = resource;
+              }
+            }
+          }
+        }
+      }
+    });
+    // console.log(amplifyApiObj)
+    let appsyncResourceObj = convertToAppsyncResourceObj(amplifyApiObj);
+    if (!_.isEmpty(this.overrideConfig) && this.overrideConfig!.overrideFlag) {
+      const overrideCode: string = fs.readFileSync(path.join(this.overrideConfig!.overrideDir, 'build', 'override.js'), 'utf-8');
+      const sandboxNode = new vm.NodeVM({
+        console: 'inherit',
+        timeout: 5000,
+        sandbox: {},
+        require: {
+          context: 'sandbox',
+          builtin: ['path'],
+          external: true,
+        },
+      });
+      try {
+        sandboxNode.run(overrideCode, path.join(this.overrideConfig!.overrideDir, 'build', 'override.js')).override(appsyncResourceObj);
+      } catch (err) {
+        const error = new Error(`Skipping override due to ${err}${os.EOL}`);
+        printer.error(`${error}`);
+        error.stack = undefined;
+        throw error;
+      }
```

Comment:
Can this logic be generalized and moved to `amplify-cli-core`?

Reply:
this package is indepedent of cli-core